app/forms/instance-create.tsx

```diff
@@ -450,7 +457,8 @@ export function CreateInstanceForm() {
         </Tabs.Content>
       </Tabs.Root>
 
-      {sourceType === 'image' && (
+      {/* siloImage and projectImage sourceTypes have additional fields */}
+      {sourceType.includes('Image') && (
```

Comment:
a less evil approach would be defining chunk of JSX up around line 188

```
const bootDiskSizeAndName = (....)
```

and then inserting it directly into the two image tabs with `{bootDiskSizeAndName}` and leaving it out of the disk one.